cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Protos/cmsAdapter.proto

```diff
@@ -258,6 +261,12 @@ message CleanPassRequest{
 	bool isCleanPass = 2;
 }
 
+message ManualPassRequest{
+	string caseId = 1;
+	bool isCleanPass = 2;
```

Comment:
Please either change this to isManualPass or (preferably) consolidate the CleanPassRequest with ManualPassRequest by using an enum for the type of change.